src/current/v24.1/migration-overview.md

```diff
@@ -62,17 +62,17 @@ A lift-and-shift approach is the most straightforward. However, it's important t
 
 - *Reduced functionality* takes some, but not all, application functionality offline. For example, you can disable writes but not reads while you migrate the application data, and queue data to be written after completing the migration.
 
-For an overview of lift-and-shift migrations to CockroachDB, see [Lift and Shift](#lift-and-shift). For considerations and details about the pros and cons of this approach, see [Migration Strategy: Lift and Shift]({% link {{ page.version.version }}/migration-strategy-lift-and-shift.md %}).
+For an overview of lift-and-shift migrations to CockroachDB, see [Lift and Shift](#lift-and-shift).
 
 #### Minimal downtime
 
-If your application cannot tolerate downtime, then you should aim for a "zero-downtime" approach. "Zero" means that downtime is reduced to either an absolute minimum or zero, such that users do not notice the migration.
+If your application cannot tolerate downtime, then you should aim for a *live migration*. This is also called a "zero-downtime" approach. "Zero" means that downtime is reduced to either an absolute minimum or zero, such that users do not notice the migration.
 
 The minimum possible downtime depends on whether you can tolerate inconsistency in the migrated data:
 
-- *Consistent* migrations reduce downtime to an absolute minimum (i.e., from 30 seconds to sub-seconds) while keeping data synchronized between the source database and CockroachDB. **Consistency requires downtime.** In this approach, downtime occurs right before [cutover](#cutover-strategy), as you drain the remaining transactions from the source database to CockroachDB.
+- Migrations with *consistent cutover* reduce downtime to an absolute minimum (i.e., from 30 seconds to sub-seconds) while keeping data synchronized between the source database and CockroachDB. **Consistency requires downtime.** In this approach, downtime occurs right before [cutover](#cutover-strategy), as you drain the remaining transactions from the source database to CockroachDB.
```

Comment:
I'm also not sure this is the clearest wording. It seems to imply that a consistent cutover will reduce downtime from 30 seconds to sub seconds. Whereas, I think in consistent cutover, the bottom line is: customers will incur downtime, but the goal is to minimize downtime to an absolute minimum. Having two downtime figures there is confusing

Reply:
Amended! TY